### DISCUSSION TITLE
放后台无任何操作，持续消耗CPU

### DISCUSSION THREAD

**Discussion by 548705486** [he3-app/he3-feedback#272] [Bug Report]
![image](https://github.com/he3-app/he3-feedback/assets/5135151/ae64e688-3521-4ee4-b3d1-154f292d32ec)

**Comment by lyzhang1999** [comment]
感谢反馈，已经记录了这个 bug 了：https://github.com/he3-app/he3-feedback/issues/277 ，我们将进一步跟进。

**Reply by VitaminC1989** [reply]
same problem
